### PR TITLE
Update Blog Contribute page.

### PR DESCRIPTION
**Pull request by aFinchy**
Signed-off-by: Amy Finch <amy@finchstudio.ca>

## Change summary

Created an updated blog contributing page in an effort to help migrate the efforts to the public and LF responsibility. 

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by aFinchy** [comment]
image

**Comment by chanmosq** [comment]
Per discussion offline with @aFinchy -
At this time, there are two PRs submitting the Blogs Process document to the [`o3de/community` repo](https://github.com/o3de/community/pull/182/) and to the Contributing Guide of the O3DE Website (this one). However, the Blogs Process documentation should live only in one location for ease of maintainability and to avoid two documents becoming out-of-sync.

This document more closely fits into the `o3de/community` repo because publishing a blog is primarily an action taken by a member of the O3DE community who coordinates with the blog author. Then, this page should be repurposed to direct folks to that document.

**Comment by aFinchy** [comment]
@chanmosq 

How would this sound on the page? 

If you're interested in writing or contributing to blogs about the Open 3D Engine, you can find a wealth of resources, including a detailed guide, templates, examples, and more, by [visiting the document located here](https://github.com/o3de/community/tree/main/markdown)

https://github.com/o3de/community/tree/main/markdown.

**Comment by aFinchy** [comment]
image

**Review by FiniteStateGit — APPROVED** [review]

File: content/docs/contributing/to-docs/blog-posts.md [hunk 1]
```diff
@@ -6,95 +6,7 @@ toc: true
...
-
-* Exaggerate or overstate your points. Choose your modifiers and adverbs with care to not oversell (or undersell) anything. Favor humble and insightful phrasing and messages over performative or provocative ones. For example, "We've seen a recent increase in users looking for a Physics solution and we're working on a solution" is a little gentler than "The world demands a Physics solution from O3DE and we're here to deliver!" Not sure how to frame your message? Talk to the community in Discord!
+Blog posts are an important and semi-formal way to deliver announcements, insights, learnings, and professional opinions for the Open 3D Engine (O3DE) organization and the greater public. Part of growing as a community is establishing a strong blog presence, and we'd love for you to contribute!
+If you're interested in writing or contributing to blogs about the Open 3D Engine, you can find a wealth of resources, including a detailed guide, templates, examples, and more, by visiting the [Markdown & Process Guides](https://github.com/o3de/community/tree/O3DE-Website/process-docs-and-markdown).
```
Comment (FiniteStateGit) [inline content/docs/contributing/to-docs/blog-posts.md:12]: The link doesn't point to a valid resource on GitHub.

File: content/docs/contributing/to-docs/blog-posts.md [hunk 2]
```diff
@@ -6,95 +6,7 @@ toc: true
 weight: 600
 ---
 
-Blog posts are an important and semi-formal way to deliver announcements, insights, learnings, and professional opinions for the **Open 3D Engine (O3DE)** organization and the greater public. Part of growing as a community is establishing a strong blog presence, and we'd love for you to contribute! Here's how.
+# Blog Process
```
Comment (FiniteStateGit) [inline content/docs/contributing/to-docs/blog-posts.md:9]: I would move this heading after any introductory information on blogs and rename it to something like "Submit a blog" or "Blog submission process".  Headings should use sentence casing.

File: content/docs/contributing/to-docs/blog-posts.md [hunk 3]
```diff
@@ -6,95 +6,7 @@ toc: true
...
-
-* Attack or criticize other products, platforms, or companies directly. Showing perf data between O3DE and a similar product/feature pairing is one thing; however, slagging on a similar product or another company is another. The latter will not be tolerated, regardless of how strongly you feel on the subject.
-
-* Exaggerate or overstate your points. Choose your modifiers and adverbs with care to not oversell (or undersell) anything. Favor humble and insightful phrasing and messages over performative or provocative ones. For example, "We've seen a recent increase in users looking for a Physics solution and we're working on a solution" is a little gentler than "The world demands a Physics solution from O3DE and we're here to deliver!" Not sure how to frame your message? Talk to the community in Discord!
+Blog posts are an important and semi-formal way to deliver announcements, insights, learnings, and professional opinions for the Open 3D Engine (O3DE) organization and the greater public. Part of growing as a community is establishing a strong blog presence, and we'd love for you to contribute!
```
Comment (FiniteStateGit) [inline content/docs/contributing/to-docs/blog-posts.md:11]: I wasn't able to view the markdown and process guides, did the content that you propose removing from the current page (e.g. _What makes a good O3DE blog post_, _Do's and Dont's_) move to a guide?

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]

File: content/docs/contributing/to-docs/blog-posts.md (hunk 1 above)
Comment (chanmosq) [reply content/docs/contributing/to-docs/blog-posts.md:12]: The link should work after this PR is merged: https://github.com/o3de/community/pull/187
Preview of doc in feature-branch: https://github.com/o3de/community/blob/Blog-Process/process-docs-and-markdown/blog-process.md

File: content/docs/contributing/to-docs/blog-posts.md (hunk 3 above)
Comment (chanmosq) [reply content/docs/contributing/to-docs/blog-posts.md:9]: I've added a comment to add that section to this guide: https://github.com/o3de/community/pull/187/files

**Review by chanmosq — APPROVED** [review]

**Review by willihay — APPROVED** [review]
Approved pending merge of https://github.com/o3de/community/pull/187

**Review by FiniteStateGit — APPROVED** [review]

**Pull request merged**